### PR TITLE
Configure Dev package source

### PR DESCRIPTION
**Pull request by agriffard**
Dedicated page to explain how to add the dev package source pointing to MyGet.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ nav:
     - About Orchard Core: docs/README.md
     - Getting started: 
         - Create a CMS Web application: docs/getting-started/README.md
+        - Configure Dev package source: docs/getting-started/devPackageSource.md
```
Comment (Piedone) [inline mkdocs.yml:64]: If we're using dashes everywhere then this should also rather be dev-package-source.md.

File: src/docs/getting-started/devPackageSource.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Add Dev package source
+
+In this article, we are going to add a new package source pointing to the Dev packages.
```
Comment (Piedone) [inline src/docs/getting-started/devPackageSource.md:3]: Some explanation on what the dev package source is wouldn't hurt. I'm new to Orchard, just reading the getting started guide. What are the dev packages? Why would I use them? What's the difference to the ones on NuGet? Should I use them in production?

File: src/docs/getting-started/devPackageSource.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## Adding Orchard Core MyGget Feed to Visual Studio
+
+In order to be able to use the __dev__ feed from Visual Studio, open the Tools menu under Nuget Package Manager --> Package Manager Settings.
```
Comment (Piedone) [inline src/docs/getting-started/devPackageSource.md:7]: It's "NuGet".

File: src/docs/getting-started/devPackageSource.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+The feed url is <https://www.myget.org/F/orchardcore-preview/api/v3/index.json>
+
+![image](assets/images/addMyGetPackageSource.png)
```
Comment (Piedone) [inline src/docs/getting-started/devPackageSource.md:10]: The image is just in the assets folder.

File: src/docs/guides/decoupled-cms/README.md [hunk 5]
```diff
@@ -324,7 +324,7 @@ We can now update the Razor Page to use the alias instead of the content item id
 }
 ```
 
-!!! waring "Release packages"
+!!! wanring "Release packages"
```
Comment (Piedone) [inline src/docs/guides/decoupled-cms/README.md:327]: I'll call Frodo to take care of the wan ring :).

Comment (agriffard) [reply src/docs/guides/decoupled-cms/README.md:327]: My precious.

File: src/docs/getting-started/devPackageSource.md (hunk 4 above)
Comment (agriffard) [reply src/docs/getting-started/devPackageSource.md:10]: ok

File: src/docs/getting-started/devPackageSource.md (hunk 3 above)
Comment (agriffard) [reply src/docs/getting-started/devPackageSource.md:7]: ok

File: src/docs/getting-started/devPackageSource.md (hunk 2 above)
Comment (agriffard) [reply src/docs/getting-started/devPackageSource.md:3]: I don't know what to say about it.

File: mkdocs.yml (hunk 1 above)
Comment (agriffard) [reply mkdocs.yml:64]: ok

File: src/docs/getting-started/devPackageSource.md (hunk 2 above)
Comment (Piedone) [reply src/docs/getting-started/devPackageSource.md]: Come on :).

**Pull request merged**